test/blackbox-tests/test-cases/glob-deps-outside-directory.t

```diff
@@ -0,0 +1,20 @@
+Glob deps in a subdirectory that refer to a path outside that subdirectory
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.6)
+  > EOF
+
+  $ mkdir -p foo bar
+
+  $ cat > foo/dune <<EOF
+  > (rule
+  >  (alias x)
+  >  (deps (glob_files ../bar/*.txt))
+  >  (action (bash "for i in %{deps}; do printf \"\$i\\n\"; done")))
```

Comment:
could you try this with `sh` instead of bash? Usually we prefer that if it works.

Reply:
Replaced `bash` with `system`